libraries/helpers.rb

```diff
@@ -55,8 +108,7 @@ def hab_binary
     end
 
     def changed_habitat_files?
-      # `changed_files` comes from the delivery-sugar DSL: https://git.io/vXvAm
-      changed_files.select { |changed_file| changed_file =~ %r{habitat/} }.any?
+      !modified_habitat_plan_contexts.empty?
```

Comment:
Nitpick: `modified_habitat_plan_contexts.any?` should be equivalent to this and avoids the double negative.